topbeat/etc/fields.yml

```diff
@@ -8,6 +8,90 @@
         Set to "system" to indicate that the statistics are system-wide.
       required: true
 
+- key: core
+  title: "CPU statistics per core"
+  description: >
+    Contains CPU statistics for each core. These statistics are the details that you can get by running the *top* command on Unix systems.
+  fields:
+    - name: id
+      type: long
+      description: >
+       The id of the core X.
+
+    - name: user
+      type: long
+      description: >
+       The amount of CPU time spent in user space on core X.
```

Comment:
If this is an amount of time, then what are the units? I believe on linux this counter increments once per every 100 cycles (100 Hz) so each increment represents 10ms.